common/config/rush/browser-approved-packages.json

```diff
@@ -2,6 +2,10 @@
 {
   "packages": [
     {
+      "name": "js-yaml",
+      "allowedCategories": [ "libraries" ]
+    },
```

Comment:
Good catch, fixed

---
In reply to: [134040545](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134040545) [](ancestors = 134040545)